client/ayon_deadline/lib.py

```diff
@@ -177,7 +177,7 @@ def _get_deadline_info(
         log.warning(f"No {item_type} retrieved")
         return []
 
-    return response.json()
+    return sorted(response.json())
```

Comment:
It would be better like this, first 'none', then rest alphabetically.
```suggestion
    return sorted(response.json(), key=lambda value: (value != "none", value))
```

Current implementation would look like this:
<img width="314" alt="chrome_4GTUeJvJsP" src="https://github.com/user-attachments/assets/c45951c9-6ec9-446c-86d0-3c794f3c3985">